CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Remove defaulting to devtool: source-map when in production with no devtool enabled. This was causing slow builds for large projects.
```

Comment:
I'm wondering if we should add docs on this in the chance someone is using this. I'm fine with the change overall.

Reply:
Pretty sure only we used these to instrument our terra-ui site, we could just add the dev-tool there.